### PR TITLE
Adds sounds to the Escape Menu

### PR DESCRIPTION
**Pull request by Sothanforax**
## About The Pull Request
https://github.com/user-attachments/assets/3e07530e-c8f1-4c21-b59b-b814831fdb2d

## Why It's Good For The Game
The aforementioned escape menu felt lacking on the sound side so with the assistance of @Sadboysuss I've added these,
they also sound pretty neat.
 
## Changelog
:cl:
sound: Added sounds for the escape menu opening/closing/being open
/:cl:

### CODE REVIEW

**Review by grungussuss — COMMENTED** [review]
sound file standards are upheld and file organization is good

**Comment by zoomachina** [comment]
god please no, hearing this loud and obnoxious sound whenever I hit escape by accident would be terrible

**Comment by Sadboysuss** [comment]
> god please no, hearing this loud and obnoxious sound whenever I hit escape by accident would be terrible

You downvoted every single sound pr I ever made until I blocked you, I don't think there's any pleasing you so if you hate all sounds, why don't you just turn off your game's volume?

**Comment by zoomachina** [comment]
> You downvoted every single sound pr I ever made until I blocked you, I don't think there's any pleasing you so if you hate all sounds, why don't you just turn off your game's volume?

I downvote bad prs, yes
the last prs you made that I disliked were #86040 and #86032 , you had to tone them down in #86835 and #87489 so clearly I had a point. It's very easy to make the game unplayable by sounds alone, and I don't want to be forced to lose audio feedback completely because some coder was eager to push their shit into the game.
Escape menu itself is just a little flair, it's not an important piece of the UI, but it's very easy to enter it by accident. Therefore, if it *has* to make a sound, it should at least not be obnoxious. This one is like the autolathe audio, and there's no "this is important machine flair!!" excuse this time.

**Comment by san7890** [comment]
i like it but i think the looping sound should fade out after 3-5 seconds so it's not always whirwhirwhirwhirwhirwhir.

**Comment by Sothanforax** [comment]
That could be done, but the intent was that this was a viewing of archival material/a cassette being played through, however if it proves to be a bit much it could lowered in volume again, or just faded out as you suggest.

**Comment by Sothanforax** [comment]
no longer a loop, things renamed and reworked accordingly with hope this is less grating overtime than a full loop.

File: code/modules/escape_menu/escape_menu.dm [hunk 1]
```diff
@@ -47,6 +47,10 @@ GLOBAL_LIST_EMPTY(escape_menus)
 	RegisterSignal(client, COMSIG_QDELETING, PROC_REF(on_client_qdel))
 	RegisterSignal(client, COMSIG_CLIENT_MOB_LOGIN, PROC_REF(on_client_mob_login))
 
+	SEND_SOUND(client, 'sound/misc/escape_menu/esc_open.ogg')
+	var/sound/esc_middle = sound('sound/misc/escape_menu/esc_middle.ogg', repeat = FALSE, channel = CHANNEL_ESCAPEMENU, volume = 80)
```
Comment (Sadboysuss) [inline code/modules/escape_menu/escape_menu.dm:51]: You need to return the code for stopping the sound on exiting the escape menu

**Comment by Sothanforax** [comment]
https://github.com/user-attachments/assets/91894495-8109-4673-9c7c-bac5e68787e3

new behavior demonstrated

**Comment by Ghommie** [comment]
I'd make the fade out begin three or two second earlier and a smidge longer but this is a good start.

**Comment by san7890** [comment]
Yeah can you show what Ghommie's suggestion would sound like? Sorry for coming back to this so late

**Comment by Sothanforax** [comment]
https://github.com/user-attachments/assets/92b84d70-89d8-45ec-9033-bb7099dc7c5f

After a brief stint where I was otherwise busy, here's the earlier fade out version

**Review by Ghommie — APPROVED** [review]

**Pull request merged**